index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+
+    <body>
+
+        <header id="header">
```

Comment:
Kindly remove all blank spaces in your code base 🙏 . For example, the blank space between the closing head tag and the opening body tag. Also, there is another blank space between the opening body tag and the opening header tag with an id of header. This will make your code clean 😍